UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Logging/EventIds.cs

```diff
@@ -856,9 +856,32 @@ public enum EventIds
         /// <summary>
         /// 805211 - Download readme.txt file is failed.
         /// </summary>
-        ErrorInDownloadReadMeFile = 805211
+        ErrorInDownloadReadMeFile = 805211,
 
+        #region V2
 
+        /// <summary>
+        /// 805212 - Request for updates since started.
+        /// </summary>
+        PostUpdatesSinceRequestStarted = 805212,
+        /// <summary>
+        /// 805213 - Request for updates since completed.
+        /// </summary>
+        PostUpdatesSinceRequestCompleted = 805213,
+        /// <summary>
+        /// 805214 - Creation of update since is started.
```

Comment:
```suggestion
        /// 805214 - Creation of updatesSince endpoint started.
```
Similarly make changes in other eventId description as well

Reply:
Resolved